src/Core/Diff.ml

```diff
@@ -0,0 +1,192 @@
+(* This file has been adapted from camlp5:
+ * https://github.com/camlp5/camlp5/blob/master/ocaml_src/lib/diff.ml
+ *)
+
+(*
+ * Copyright (c) 2007-2017, INRIA (Institut National de Recherches en
+ * Informatique et Automatique). All rights reserved.
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
```

Comment:
This part implies to me that we might need to add it to `ThirdPartyLicenses` (and perhaps surface that in the docs?).

Reply:
Yeah, I think you're right. Added in https://github.com/onivim/oni2/pull/1240/commits/2626a6cb56fe6ab4790889759e919f050b99dc42

Reply:
Perfect! Thanks for adding it.